xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsDisplayerUIX.xml

```diff
@@ -797,7 +800,7 @@ require(['jquery', 'xwiki-meta', 'xwiki-bootstrap-switch'], function ($, xm) {
  * Notifications Header
  ***********************************************/
 #tmNotifications {
-  &amp; > button {
+  &amp; &gt; button {
```

Comment:
It means the rule introduced in https://github.com/xwiki/xwiki-platform/commit/da56c0defbd5d810035ce9164dd38ad08984a709#diff-cf46b5e440697c137f189087169625042ba5cc5ad1d675ddbe82f46d7c2aa703 wasn't working?

Reply:
Yes, escaping `>` is not required, but it's done with the XML serializer we are using in the xar format plugin.